src/Data/Nat/Binary/Properties.agda

```diff
@@ -1061,7 +1075,7 @@ private
   }
 
 *-1-monoid : Monoid 0ℓ 0ℓ
-*-1-monoid = record
+*-1-monoid   = record
```

Comment:
These equal signs should not be aligned. Same with `*-+-semiring` below.

Reply:
Fixed.